DotBot.py

```diff
@@ -37,6 +37,8 @@ def __init__(self, dotBotId, x, y, floorplan):
         self.next_bump_x          = None  # coordinate the DotBot will bump into next
         self.next_bump_y          = None
         self.next_bump_ts         = None  # time at which DotBot will bump
+        self.newMovementReceived  = True
+        self.notificationID       = 0
```

Comment:
this isn't an identifier, it's a sequence number. Change name to `seqNumNotification`